roles/lib_openshift/src/doc/service

```diff
@@ -0,0 +1,121 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_service
+short_description: Create, modify, and idempotently manage openshift services.
+description:
+  - Manage openshift service objects programmatically.
+options:
+  state:
+    description:
+    - State represents whether to create, modify, delete, or list
+    required: true
+    default: present
+    choices: ["present", "absent", "list"]
+    aliases: []
+  kubeconfig:
+    description:
+    - The path for the kubeconfig file to use for authentication
+    required: false
+    default: /etc/origin/master/admin.kubeconfig
+    aliases: []
+  debug:
+    description:
+    - Turn on debug output.
+    required: false
+    default: False
+    aliases: []
+  name:
+    description:
+    - Name of the object that is being queried.
+    required: false
+    default: None
+    aliases: []
+  namespace:
+    description:
+    - The namespace where the object lives.
+    required: false
+    default: default
+    aliases: []
+  selector:
+    description:
+    - The selector to apply when filtering for services.
+    required: false
+    default: None
+    aliases: []
+  labels:
+    description:
+    - The labels to apply on the service.
+    required: false
+    default: None
+    aliases: []
+  clusterip:
+    description:
+    - The cluster ip address to use with this service.
+    required: false
+    default: None
+    aliases: []
+  portalip:
+    description:
+    - The portal ip address to use with this service.
```

Comment:
Virtual IP of the service, allocated automatically at creation from a pool of internal IPs.